### ISSUE TITLE
Include build information in the Demo

### ISSUE THREAD

**Issue by sdasda7777** [emilk/egui#4215]
**Is your feature request related to a problem? Please describe.**
I think it would be a good idea to show build information somewhere in the demo. Especially since the Demo seems to be considered a source of truth by some. 

**Describe the solution you'd like**
Just a label or something that would display egui version and date of the given build.

**Additional context**
#4213

**Comment by sdasda7777** [comment]
Also, the Demo build scripts are compiling with no output, making it seem like nothing is happening 👎

**Comment by emilk** [comment]
> Also, the Demo build scripts are compiling with no output, making it seem like nothing is happening 👎

Work on my machine:

```
❯ ./scripts/build_demo_web.sh 
info: component 'rust-std' for target 'wasm32-unknown-unknown' is up to date
Building rust…
Generating JS bindings for wasm…
Finished web_demo/egui_demo_app_bg.wasm
```

**Comment by sdasda7777** [comment]
> Work on my machine:

Yeah, I mean the `Building rust...` step. I thought it was broken because it had no other output for a few minutes, so I killed it, but it turns out it would have completed if I let it run few minutes longer 🤷‍♂️

**Comment by YgorSouza** [comment]
> Yeah, I mean the `Building rust...` step. I thought it was broken because it had no other output for a few minutes, so I killed it, but it turns out it would have completed if I let it run few minutes longer 🤷‍♂️

That's actually just a matter of removing this line:

https://github.com/emilk/egui/blob/21835c31760f0c3e03d0d3db1f44b00ecb955734/scripts/build_demo_web.sh#L90
